cmd/utils.go

```diff
@@ -617,7 +617,7 @@ func getConfigLabels(projectConfig ProjectConfig) (map[string]string, error) {
 	}
 
 	if projectConfig.License != "" {
-		if valid, err := IsValidKubernetesLabelValue(projectConfig.License); !valid {
+		if valid, err := IsValidKubernetesLabelValue(projectConfig.License); !valid || !checkValidLicense(projectConfig.License) {
 			return labels, errors.Errorf("%s license value is invalid. %v", ConfigFile, err)
```

Comment:
We need to have a better error message here if the license doesn't match an SPDX, otherwise the user will not know how to fix the error. I think we should link to a URL with all the valid values.

Reply:
Sandy, this is almost perfect, but there's a small issue with the error message. The getConfigLabels() method is called from both the stack package (using stack.yaml license) and from the build command (using .appsody-config.yaml). Thus, the error message cannot say "stack must have". Ideally we should tell the user exactly what file has the bad license.